ppl/cmd/testtoken/testtoken.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/brimsec/zq/ppl/zqd/auth"
+	"github.com/mccanne/charm"
+)
+
+var CLI = &charm.Spec{
+	Name:  "testtoken",
+	Usage: "testtoken",
+	Short: "generate access token to test zqd auth",
```

Comment:
Nit: testtoken reads like it tests tokens rather than creates them. Maybe call this gentoken or maketoken?

Reply:
👍 , will use 'gentoken'